R/predict_amps.R

```diff
@@ -24,6 +24,10 @@
 
 predict_amps <- function(faa_df, min_len = 5, n_cores=1, model = "precursor") {
 
+  if (!is.character(faa_df[,2])){
+    stop("Sequences are required to be in character format")
```

Comment:
How about giving the user more info to help them debug this

```suggestion
    col2format <- class(faa_df[,2])
    stop(paste("Sequences are required to be in character format but are provided as ",col2format))
```